src/main/java/edu/cornell/kfs/fp/document/CuDisbursementVoucherDocument.java

```diff
@@ -613,13 +614,14 @@ protected DocumentHelperService getDocumentHelperService() {
         return documentHelperService;
     }
 
-    public CuDisbursementVoucherPayeeDetail getDvPayeeDetail() {
+    @Override
+    public DisbursementVoucherPayeeDetail getDvPayeeDetail() {
```

Comment:
Now that the dvPayeeDetail variables have been consolidated, this getter seems to be the same as in the superclass and isn't adding any value anymore. I would suggest removing it.

If you would rather defer this cleanup to a future user story so that we can keep these changes moving forward, please let me know.

Reply:
This getter has been removed. Doing the cleanup now reduces the burden of validation testing for the users.